helpers/file_helper.rb

```diff
@@ -19,4 +19,13 @@ def self.file_rename(file_name)
     File.rename("files_tmp/#{file_name}", "files_tmp/#{file_new_name}")
     file_new_name
   end
+
+  def self.dir_check_or_create(dir_name)
+    if File.exist? dir_name.to_s
+      puts 'files_tmp exist: true'
+    else
+      Dir.mkdir dir_name.to_s
+    end
+    true
```

Comment:
I don't think there is a need to return true value, since method handle both possible cases